examples/ZMP/ZMPCachedLQR.m

```diff
@@ -0,0 +1,41 @@
+function [K, S] = ZMPCachedLQR(h_over_g, Qy)
+% NOTEST
+
+MAX_CACHE_ENTRIES = 1000;
+
+key = [sprintf('hg %f ', h_over_g), 'Qy ', sprintf('%f', Qy)];
+
+persistent lqr_solution_cache
+if isempty(lqr_solution_cache)
+  fname = fullfile(getDrakePath(), 'examples', 'ZMP', 'data', 'lqr_solution_cache.mat');
```

Comment:
@rdeits says the first call takes 10 seconds slower.  must be loading in some toolbox in matlab.